tests/lts_compatibility.py

```diff
@@ -48,6 +48,38 @@ def get_bin_and_lib_dirs_for_install_path(install_path):
     )
 
 
+def set_js_args(args, from_install_path):
+    # Use from_version's app and constitution as new JS features may not be available
+    # on older versions
+    js_app_directory = (
+        "../samples/apps/logging/js"
+        if from_install_path == LOCAL_CHECKOUT_DIRECTORY
+        else "samples/logging/js"
+    )
+    args.js_app_bundle = os.path.join(from_install_path, js_app_directory)
+
+    constitution_directory = os.path.join(
+        from_install_path,
+        "../src/runtime_config/default"
+        if from_install_path == LOCAL_CHECKOUT_DIRECTORY
+        else "bin",
+    )
+
+    def replace_constitution_fragment(args, fragment_name):
+        args.constitution[:] = [
+            os.path.join(constitution_directory, fragment_name)
+            if fragment_name in f
+            else f
+            for f in args.constitution
+        ]
+
+    # Note: Use resolve.js script from local checkout as only the trivial sandbox
+    # version is included in installation
```

Comment:
I'm in favour of shipping the default `resolve.js` script as well as the pass-through sandbox one. Any objections?